README.md

```diff
@@ -296,6 +292,8 @@ Are you interested in contributing to external-dns? We, the maintainers and comm
 suggestions, contributions, and help! Also, the maintainers can be contacted at any time to learn more
 about how to get involved.
 
+End-to-end testing of ExternalDNS is currently [performed](https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/test/e2e/external_dns.go) in the separate [kubernetes-on-aws](https://github.com/zalando-incubator/kubernetes-on-aws) repository.
+
```

Comment:
I think we should get rid of this line. We have zero access to Zalando's infra and this was mostly a line that was there when the project was 100% developed at Zalando, which is not the case anymore. I have some e2e that I run for releases, which are closed source and that I don't have intention to upstream / open due to how they have been written, but there are also tests written by Azure, that we may integrate more visibly in the process. I think for now let's remove this line, we'll figure later on how to provide info on what is being tested and how.

Reply:
Sounds good to me. I updated the PR.